### PR TITLE
Upgrade skywalking-cli dependency

### PR DESCRIPTION
**Pull request by hanahmily**

### CODE REVIEW

File: pkg/provider/provider.go [hunk 1]
```diff
@@ -310,3 +296,24 @@ func (p *externalMetricsProvider) selectGroupResource(namespace string) apischem
 func (p *externalMetricsProvider) getMetricNameWithNamespace(metricName string) string {
 	return strings.Join([]string{p.namespace, metricName}, "|")
 }
+
+// TODO: remove this function once cli move it from internal module to pkg
+func parseScope(entity *swctlapi.Entity) swctlapi.Scope {
```
Comment (hanahmily) [inline pkg/provider/provider.go:301]: @kezhenxu94 this func is copied from cli repo. Could we move it from internal to pkg?

File: pkg/provider/provider.go [hunk 2]
```diff
@@ -310,3 +296,24 @@ func (p *externalMetricsProvider) selectGroupResource(namespace string) apischem
 func (p *externalMetricsProvider) getMetricNameWithNamespace(metricName string) string {
 	return strings.Join([]string{p.namespace, metricName}, "|")
 }
+
+// TODO: remove this function once cli move it from internal module to pkg
```
Comment (wu-sheng) [inline pkg/provider/provider.go:300]: Do we have an issue tracking this?

Comment (hanahmily) [reply pkg/provider/provider.go:300]: on it.

File: pkg/provider/provider.go (hunk 1 above)
Comment (kezhenxu94) [reply pkg/provider/provider.go:301]: > @kezhenxu94 this func is copied from cli repo. Could we move it from internal to pkg?

@fgksgf can you take over this? Thanks

Comment (fgksgf) [reply pkg/provider/provider.go:301]: @kezhenxu94 No problem, i will do this today

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**